test/functional/tests.go

```diff
@@ -7,6 +7,7 @@ var (
 		{Description: "F01 - Verify AWS rds resources exist and are in expected state", Test: AWSRDSResourcesExistTest},
 		{Description: "F03 - Verify AWS elasticache resources exist and are in expected state", Test: AWSElasticacheResourcesExistTest},
 		{Description: "A21 - Verify AWS maintenance and backup windows", Test: CROStrategyOverrideAWSResourceTest},
+		{Description: "Verify AWS Standalone VPC exists", Test: TestStandaloneVPCExists},
```

Comment:
We should probably make sure this has an ID/code associated with it in the test cases

Reply:
Sure, @psturc is there a test case already associated to this?

Reply:
@dimitraz I'll sent a PR for the test case and ping you on it